include/crocoddyl/multibody/fwd.hpp

```diff
@@ -299,6 +299,10 @@ typedef CostDataFrameRotationTpl<double> CostDataFrameRotation;
 typedef ImpulseModelAbstractTpl<double> ImpulseModelAbstract;
 typedef ImpulseDataAbstractTpl<double> ImpulseDataAbstract;
 
+enum ContactType { Contact3D, Contact6D, ContactUndefined };
+
+enum ImpulseType { Impulse3D, Impulse6D, ImpulseUndefined };
```

Comment:
I don't think this one is really needed.
The contact can be either 2D, 3D, 6D, etc. and can be solved using a Force, Impulse or Impact scheme.

Reply:
You are right, we don't need these two separate enums. Currently there is additional support of Contact2D in another PR, while 2D impulse is not yet supported. I need to add support of Impulse2D as well, and then I can merge these two together.